library/src/main/java/io/appwrite/services/KeepAliveService.kt

```diff
@@ -0,0 +1,14 @@
+package io.appwrite.services
```

Comment:
How do we use this class?

Reply:
This ensures that the calling activity is kept alive while the OAuth process happens in a custom tab